docs/bluemix-postgres.md

```diff
@@ -108,18 +119,40 @@ Congratulation. Now you can use the link **http://[IP]:30080** to access your gi
 
 > Note: For the above example, the link would be http://169.47.241.106:30080  since its IP is `169.47.241.106` and the UI port number is `30080`.
 
+Depending on your cluster, the output might be as follows:
+
+```bash
+$ kubectl get nodes -o wide
+NAME             STATUS    ROLES     AGE       VERSION       EXTERNAL-IP     OS-IMAGE             KERNEL-VERSION      CONTAINER-RUNTIME
+10.176.239.136   Ready     <none>    2d        v1.10.8+IKS   169.47.252.83   Ubuntu 16.04.5 LTS   4.4.0-137-generic   docker://17.6.2
+10.176.239.146   Ready     <none>    2d        v1.10.8+IKS   169.47.252.51   Ubuntu 16.04.5 LTS   4.4.0-137-generic   docker://17.6.2
+10.176.239.161   Ready     <none>    2d        v1.10.8+IKS   169.47.252.52   Ubuntu 16.04.5 LTS   4.4.0-137-generic   docker://17.6.2
+$ kubectl get svc gitlab
+NAME      TYPE       CLUSTER-IP       EXTERNAL-IP   PORT(S)                     AGE
+gitlab    NodePort   172.21.170.161   <none>        80:30080/TCP,22:30022/TCP   2d
+```
+
+> Note: In this situation, you need to identify the gitlab node in the Kubernetes UI. Check the "Node" IP for the gitlab pod abd use this to get the external IP in the above command. 
+
+> Note: For the above example, the link would be http://169.47.252.51:30080  since its node IP is `10.176.239.146` and therfore the external IP is `169.47.252.51` and the UI port number is `30080`.
```

Comment:
this is a duplicate note from above?

Reply:
sorry, I meant rather from 10 lines above: " Note: For the above example, the link would be http://169.47.241.106:30080  since its IP is `169.47.241.106` and the UI port number is `30080`."